### ISSUE TITLE
Parsed tx.nonce returns empty value for nonce=0

### ISSUE THREAD

**Issue by max-block** [ethereumjs/ethereumjs-vm#714]
There are two raw txs:
- tx1 with nonce=0
- tx2 with nonce=3

Let's parse them:
```
const Tx = require("ethereumjs-tx")
const tx1 = new Tx(
  "0xf86c808502cb417800827148948bc1779b4cb3fd0b33a3b2c129a1178f5b439ae688120a871cc00200008026a07b77ce4bc7d8b16a8b0fc2407262ffe4e365fefc7d49bb9777938628cdc2c1d2a00d01ec810908f69be06bfbe805dcf193804c2f5e4ca664b714436e046797f4e4"
)
const tx2 = new Tx(
  "0xf86c038502cb4178008271489448797ee0870a386c66dd8b0c5a233b01566be16288120a871cc00200008026a02428df35c47b2c5f671e31a65dc8f2ee287a805236c2a870e430f7811f15accea0293044614b92837228219ea7418b427c34aa2f6bfa2f1619f4d634a80d449338"
)
console.log(`tx1.nonce(${tx1.nonce.toString("hex")}), length=${tx1.nonce.length}`)
console.log(`tx2.nonce(${tx2.nonce.toString("hex")}), length=${tx2.nonce.length}`)
```

It prints:
```
tx1.nonce(), length=0
tx2.nonce(03), length=1
```

So for nonce=0 it returns empty value, but it should returns 0.

**Comment by holgerd77** [comment]
Hi @ryanio, can you have a look along your work on #812 if this is still an issue?

**Comment by holgerd77** [comment]
Update: depending on the answer here please also have a short look on the state of #712, since it was suggested in the last comment that the issues might be connected. Thanks!

**Comment by ryanio** [comment]
This should be resolved with #812 since `nonce` will be stored as a `BN` instead of a `Buffer`, and `stripZeroes` is no longer used.
